network/ovs_endpointclient_linux.go

```diff
@@ -157,35 +155,27 @@ func (client *OVSEndpointClient) AddEndpointRules(epInfo *EndpointInfo) error {
 }
 
 func (client *OVSEndpointClient) DeleteEndpointRules(ep *endpoint) {
-	logger.Info("[ovs] Get ovs port for interface", zap.String("HostIfName", ep.HostIfName))
+	logger.Info("[ovs] Delete endpoint rules", zap.String("HostIfName", ep.HostIfName))
 	containerPort, err := client.ovsctlClient.GetOVSPortNumber(client.hostVethName)
 	if err != nil {
 		logger.Error("[ovs] Get portnum failed with", zap.Error(err))
 	}
 
-	logger.Info("Get ovs port for interface", zap.String("hostPrimaryIfName", client.hostPrimaryIfName))
 	hostPort, err := client.ovsctlClient.GetOVSPortNumber(client.hostPrimaryIfName)
 	if err != nil {
 		logger.Error("Get portnum failed with", zap.Error(err))
 	}
 
 	// Delete IP SNAT
-	logger.Info("[ovs] Deleting IP SNAT for port", zap.String("containerPort", containerPort))
 	client.ovsctlClient.DeleteIPSnatRule(client.bridgeName, containerPort)
 
 	// Delete Arp Reply Rules for container
-	logger.Info("[ovs] Deleting ARP reply rule for ip vlanid for container port", zap.String("address", ep.IPAddresses[0].IP.String()),
-		zap.Any("VlanID", ep.VlanID), zap.String("containerPort", containerPort))
 	client.ovsctlClient.DeleteArpReplyRule(client.bridgeName, containerPort, ep.IPAddresses[0].IP, ep.VlanID)
 
 	// Delete MAC address translation rule.
-	logger.Info("[ovs] Deleting MAC DNAT rule for IP address and vlan", zap.String("address", ep.IPAddresses[0].IP.String()),
-		zap.Any("VlanID", ep.VlanID))
 	client.ovsctlClient.DeleteMacDnatRule(client.bridgeName, hostPort, ep.IPAddresses[0].IP, ep.VlanID)
 
 	// Delete port from ovs bridge
-	logger.Info("[ovs] Deleting MAC DNAT rule for IP address and vlan", zap.String("address", ep.IPAddresses[0].IP.String()),
-		zap.Any("VlanID", ep.VlanID))
 	if err := client.ovsctlClient.DeletePortFromOVS(client.bridgeName, client.hostVethName); err != nil {
 		logger.Error("[ovs] Deletion of interface from bridge failed", zap.String("hostVethName", client.hostVethName), zap.String("bridgeName", client.bridgeName))
```

Comment:
This might be to heavy handed. Need to look at logs to see what we would lose if we delete all of these logs.